code/src/components/Main.js

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable no-underscore-dangle */
+import React, { useState, useEffect } from 'react';
+import GetThoughts from './GetThoughts';
+import UserInput from './UserInput';
+
+const Main = () => {
+  const [getThoughts, setGetThoughts] = useState([]);
+  const [newThought, setNewThought] = useState('');
+  /* const [loading, setLoading] = useState(false); */
+
+  const APIurl = 'https://happy-thoughts-technigo.herokuapp.com/thoughts'
```

Comment:
Really good job with Const APIurl. You have a very clean and nice code. I also need to get better to use API in this way.